components/properties/Cargo.toml

```diff
@@ -38,6 +38,7 @@ icu_provider = { version = "0.5", path = "../../provider/core", features = ["mac
 icu_uniset = { version = "0.4.1", path = "../../utils/uniset"}
 serde = { version = "1.0", default-features = false, features = ["derive", "alloc"], optional = true }
 zerovec = { version = "0.6", path = "../../utils/zerovec", features = ["derive"] }
+unicode-bidi = "0.3.7"
```

Comment:
Issue: This should be an optional dependency. Enabling this dependency should enable the code that uses it.

Reply:
done!